internal/oc/release.go

```diff
@@ -396,12 +393,7 @@ func (r *release) extractFromRelease(log logrus.FieldLogger, releaseImage, cache
 		return "", err
 	}
 
-	var cmd string
-	if icspFile == "" {
-		cmd = fmt.Sprintf(templateExtract, binary, workdir, insecure, releaseImage)
-	} else {
-		cmd = fmt.Sprintf(templateExtractWithIcsp, binary, workdir, insecure, icspFile, releaseImage)
-	}
+	cmd := fmt.Sprintf(templateExtract, binary, workdir, insecure, mirrorsFlag, releaseImage)
```

Comment:
Sorry, I've never seen it like this before, but does `Sprintf` expand `mirrorsFlag`? In other words, you don't need to do 
```
cmd := fmt.Sprintf(templateExtract, binary, workdir, insecure, mirrorsFlag.String(), releaseImage)
```

Reply:
No, that explicit conversion isn't needed, the printf-like strings do that conversion automatically. From https://pkg.go.dev/fmt:

> 5. If an operand implements method String() string, that method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

Se an example here: https://go.dev/play/p/3m2nGM1qsIc

Reply:
Very nice, thanks for the explanation!